README.md

```diff
@@ -3,8 +3,6 @@
   <h1 align="center">nexus-prisma</h1>
 </p>
 
-[![CircleCI](https://circleci.com/gh/graphql-nexus/nexus-prisma.svg?style=svg)](https://circleci.com/gh/graphql-nexus/nexus-prisma)
```

Comment:
Anyway we could bring back the badge to use gh actions?

Reply:
https://prisma-company.slack.com/archives/C8AKVD5HU/p1585238227035900